inference-engine/tests/functional/inference_engine/CMakeLists.txt

```diff
@@ -54,6 +54,12 @@ if(NGRAPH_ONNX_IMPORT_ENABLE)
     add_dependencies(${TARGET_NAME} inference_engine_onnx_reader)
 endif()
 
+if(NGRAPH_PDPD_FRONTEND_ENABLE)
+    target_compile_definitions(${TARGET_NAME} PRIVATE
+        NGRAPH_PDPD_FRONTEND_ENABLE
```

Comment:
is this macro necessary?

Reply:
It is done for onnx reader, you think it isn't needed in this case?

Reply:
If you want to have "#ifdef NGRAPH_PDPD_FRONTEND_ENABLE" in the code - then this macro is needed. If there is no such code now - I suggest to remove it from here
ONNX has "#ifdef NGRAPH_ONNX_IMPORT_ENABLE" in test code, thus they need it

Reply:
Done